### PR TITLE
Make ONB default

### PR DESCRIPTION
**Pull request by mateuszbaran**
I think this would be nice for convenience.

### CODE REVIEW

File: src/bases.jl [hunk 1]
```diff
@@ -817,13 +839,16 @@ function get_vector_cached!(M::AbstractManifold, Y, p, X, B::CachedBasis)
 end
 
 """
-    get_vectors(M::AbstractManifold, p, B::AbstractBasis)
+    get_vectors(M::AbstractManifold, p, B::AbstractBasis=get_basis(M, p, DefaultOrthonormalBasis()))
```
Comment (kellertuer) [inline src/bases.jl:842]: Why do the others just need `DefaultOrthonormalBasis()` but this here the same wrapped in another call? And the code seems to do something different, namely not call `get_basis`?

Comment (mateuszbaran) [reply src/bases.jl:842]: `get_vectors` is generally supposed to work with cached bases, so we need to get a cached basis first.

Comment (kellertuer) [reply src/bases.jl:842]: Then we could maybe mention that in line 844 in the docs here?

Comment (mateuszbaran) [reply src/bases.jl:842]: Sure, I've added a note there.

**Pull request merged**